### PR TITLE
[JUnit] Introduce the property of trimStackTrace to show error stacktrace in mvn-test

### PR DESCRIPTION
**Pull request by zuston**
### What changes were proposed in this pull request?
[JUnit] Introduce the property of trimStackTrace to show error stacktrace in mvn-test

### Why are the changes needed?
When acting mvn test in current codebase, it will not throw detailed stacktrace about failed UTs and make hard to debug. So introduce this property of trimStackTrace to allow user to show the complete stacktrace. 

### Does this PR introduce _any_ user-facing change?
NO

### CI Before this PR
1. https://github.com/zuston/incubator-uniffle/actions/runs/2800975455#artifacts
image

### After 
1. https://github.com/apache/incubator-uniffle/runs/7684126515?check_suite_focus=true
image

### CODE REVIEW

**Comment by jerqi** [comment]
Could you show me the effect of this plugin by you own Github action? like #122

**Comment by jerqi** [comment]
If you want to see the log, you can download https://github.com/apache/incubator-uniffle/actions/runs/2779258464 
企业微信截图_ddffa6a0-c008-4150-bfa7-3fa21a08d420
Click the link which I give you, and Search `Artifacts`

**Comment by zuston** [comment]
The failed test cases stracktrace will be shown in artifacts? I didn’t find. 

And i think it’s meaningful to allow user to specify the config to control the stacktrace output.

**Comment by jerqi** [comment]
> The failed test cases stracktrace will be shown in artifacts? I didn’t find.
> 
Yes, Click the link which I give you, and Search Artifacts, you can find it.

**Comment by zuston** [comment]
Recheck the log in artifact, still not found the junit test case error detailed stacktrace.

**Comment by jerqi** [comment]
> Could you show me the effect of this plugin by you own Github action? like #122

Could you give me an example?

**Comment by zuston** [comment]
The default value of trimStackTrace is true, which will not output the error stackstrace. But when we execute the command of `mvn clean install -Pspark3 -DskipTests -DtrimStackTrace=true`, it will be valid.

> Could you give me an example?

Ok, it will change the default value to false.

**Comment by zuston** [comment]
@jerqi Like this https://github.com/apache/incubator-uniffle/runs/7684126515?check_suite_focus=true

Screen Shot 2022-08-05 at 10 26 50 AM

**Comment by jerqi** [comment]
The artifact log which  I give you have these logs, you can find it in *.out.

**Comment by zuston** [comment]
Due to this PR, so the detailed log can be shown in the console and output log.

**Comment by jerqi** [comment]
> Due to this PR, so the detailed log can be shown in the console and output log.

I don't use your pr.

**Comment by zuston** [comment]
### CI Before this PR
1. https://github.com/zuston/incubator-uniffle/actions/runs/2800975455#artifacts
image

### After 
1. https://github.com/apache/incubator-uniffle/runs/7684126515?check_suite_focus=true
image

**Comment by zuston** [comment]
Updated @jerqi

File: integration-test/common/pom.xml [hunk 1]
```diff
@@ -144,6 +144,7 @@
                         <useFile>${test.redirectToFile}</useFile>
                         <argLine>-ea -Xmx3g</argLine>
                         <failIfNoTests>false</failIfNoTests>
+                        <trimStackTrace>${trimStackTrace}</trimStackTrace>
```
Comment (jerqi) [inline integration-test/common/pom.xml:147]: Why do we need this? Is it enough to change parent pom file? And why do we only modify common, mr, spark-common? How about spark2 and spark3?

Comment (zuston) [reply integration-test/common/pom.xml]: Tested. The trimStackTrace in parent pom is enough.

**Review by jerqi — APPROVED** [review]
LGTM, thanks @zuston

**Pull request merged**